bitnami/kube-prometheus/templates/prometheus/prometheus.yaml

```diff
@@ -183,10 +183,15 @@ spec:
   {{- if .Values.prometheus.volumeMounts }}
   volumeMounts: {{- include "common.tplvalues.render" (dict "value" .Values.prometheus.volumeMounts "context" $) | nindent 4 }}
   {{- end }}
-  {{- if .Values.prometheus.additionalScrapeConfigsExternal.enabled }}
+  {{- if .Values.prometheus.additionalScrapeConfigs.enabled }}
```

Comment:
What is about keeping retro compatibility falling back to the old values so we don't need to break it?

We trying to avoid many major versions in the charts.

```
if or .Values.prometheus.additionalScrapeConfigs.enabled .Values.prometheus.additionalScrapeConfigsExternal.enabled
...
```

Reply:
@dani8art Sure, how would you suggest this be noted in the docs? Would you put a note next to the value that it's deprecated?